visualization/app/codeCharta/state/store/appSettings/isHeightAndColorMetricLinked/isHeightAndColorMetricLinked.reducer.ts

```diff
@@ -0,0 +1,14 @@
+import {
+	defaultIsHeightAndColorMetricLinked,
+	IsHeightAndColorMetricLinkedAction,
+	IsHeightAndColorMetricLinkedActions
+} from "./isHeightAndColorMetricLinked.actions"
+
+export function isHeightAndColorMetricLinked(state = defaultIsHeightAndColorMetricLinked, action: IsHeightAndColorMetricLinkedAction) {
+	switch (action.type) {
+		case IsHeightAndColorMetricLinkedActions.TOGGLE_IS_HEIGHT_AND_COLOR_METRIC_LINKED:
```

Comment:
[Optional] although this is just a simple "if else", it would be easy to write a quick unit test for it :)